pylearner.ts

```diff
@@ -1026,6 +1050,29 @@ class SelectExpression extends Expression {
     this.push(bindingThunk(pop).value);
   }
 }
+class AppendExpression extends Expression {
+  constructor(loc: Loc, instrLoc: Loc, public target: Expression, public item: Expression) {
+    super(loc, instrLoc);
+  }
+
+  check(env: Scope) {
+    let targetType = this.target.check_(env);
+    if (!(targetType.isListType()))
+      this.executionError("Het doel van een append-uitdrukking moet een lijst zijn");
+    this.item.checkAgainst(env, intType);
+    return targetType;
+  }
+
+  async evaluate(env: Scope) {
+    await this.target.evaluate(env);
+    await this.item.evaluate(env);
+    let [target, item] = pop(2);
```

Comment:
Voeg een breakpoint in net vóór het poppen, zodat de Small step-knop stopt na het evalueren van de subexpressies maar vóór de uitvoering van de methode.

```suggestion
    await this.breakpoint();
    let [target, item] = pop(2);
```

Reply:
Breakpoint is toegevoegd.